src/com/lftechnology/java/training/alina/collections/generics/instantiateproduct/Car.java

```diff
@@ -0,0 +1,20 @@
+
+package com.lftechnology.java.training.alina.collections.generics.instantiateproduct;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Car Class consists of simple log level description of car
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class Car {
+
+	private static final Logger LOGGER = Logger.getLogger(Car.class.getName());
+
+	public Car() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Hide this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)